components/input.css

```diff
@@ -0,0 +1,48 @@
+.input-box {
+    padding: 0.5rem 1rem;
+    margin: 0.5rem 0;
```

Comment:
Something like 0.5rem is quite small in my opinion when calculated in absolute value. So you could give them absolute values directly, as giving them such a low value in rems won't change them much on smaller screens also.